build.gradle

```diff
@@ -33,24 +35,32 @@ ext {
       "org.codehaus.groovy:groovy-test:${groovyVersion}",
       "org.codehaus.groovy:groovy-sql:${groovyVersion}",
       "org.codehaus.groovy:groovy-xml:${groovyVersion}",
+      "org.codehaus.groovy:groovy-jaxb:${groovyVersion}",
     ]
   } else {
     throw new InvalidUserDataException("Unknown variant: $variant. Choose one of: $variants")
   }
+
   maxGroovyVersion = snapshotVersion ? "3.9.99" : "2.9.99" // TODO lock these down once Groovy 3 gets close to RC stage
   if (System.getProperty("groovyVersion")) {
     groovyVersion = System.getProperty("groovyVersion")
   }
   fullVersion = baseVersion + ((!snapshotVersion && releaseCandidate) ? "-RC$releaseCandidate" : "") + "-groovy-$variant" + (snapshotVersion ? "-SNAPSHOT" : '')
   variantLessVersion = baseVersion + (snapshotVersion ? "-SNAPSHOT" : (releaseCandidate ? "-RC$releaseCandidate" : ""))
-  javaVersions = [1.7, 1.8]
+  javaVersions = [1.8, 9, 10, 11]
```

Comment:
Spock 1.x should still support Java 7, it should work if you add it back in

Reply:
Fixed.